generator.yml

```diff
@@ -59,13 +59,13 @@ defconfigs:
   - &arm64_gki  {config: gki_defconfig,                                << : *arm64-triple,         << : *kernel_modules}
   - &arm64_cut  {config: cuttlefish_defconfig,                         << : *arm64-triple,         << : *kernel_modules}
   - &i386       {config: defconfig,                                    << : *i386-triple,          << : *kernel_modules}
-  - &mips       {config: [malta_kvm_guest_defconfig, CONFIG_CPU_BIG_ENDIAN=y],                    << : *mips-triple,          << : *kernel_modules}
-  - &mipsel     {config: malta_kvm_guest_defconfig,                    << : *mipsel-triple,        << : *kernel_modules}
-  - &ppc32      {config: ppc44x_defconfig,                             << : *powerpc-triple,       << : *kernel_modules}
-  - &ppc64      {config: pseries_defconfig,                            << : *powerpc64-triple,     << : *kernel_modules}
-  - &ppc64le    {config: powernv_defconfig,                            << : *powerpc64le-triple,   << : *kernel_modules}
+  - &mips       {config: [malta_kvm_guest_defconfig, CONFIG_CPU_BIG_ENDIAN=y], kernel_image: vmlinux, << : *mips-triple,          << : *kernel_modules}
+  - &mipsel     {config: malta_kvm_guest_defconfig,                    kernel_image: vmlinux, << : *mipsel-triple,        << : *kernel_modules}
+  - &ppc32      {config: ppc44x_defconfig,                             kernel_image: uImage, << : *powerpc-triple,       << : *kernel_modules}
+  - &ppc64      {config: pseries_defconfig,                            kernel_image: vmlinux, << : *powerpc64-triple,     << : *kernel_modules}
+  - &ppc64le    {config: powernv_defconfig,                            kernel_image: zImage.epapr, << : *powerpc64le-triple,   << : *kernel_modules}
   #                                  https://github.com/ClangBuiltLinux/linux/issues/1143
-  - &riscv      {config: [defconfig, CONFIG_EFI=n],                    << : *riscv-triple,         << : *kernel_modules}
+  - &riscv      {config: [defconfig, CONFIG_EFI=n],                    kernel_image: Image, << : *riscv-triple,         << : *kernel_modules}
```

Comment:
Does specifying a `kernel_image` to tuxsuite prevent the kernel modules from being built?  (We don't want that; we want to build the kernel modules).  cc @danrue @terceiro

Reply:
No. The modules are a separate target and we always build those.